### DISCUSSION TITLE
Integrating with Active Directory

### DISCUSSION THREAD

**Discussion by jedilx** [balzack/databag#27] [Ideas]
Hello friends!
Can you advise if it's possible to use this application to integrate with Active Directory, for example, to reset a user's password in AD? So that users can access AD through it?
If anyone has any ideas, please share them.

**Comment by balzack** [comment]
Databag currently does not integrate with AD. My view of databag is more of a personal tool than an enterprise tool, so that feature is not a high priority. Please let me know if you disagree with that or my view of AD.

**Reply by ghost** [reply]
I agree. 
I plan to use it on raspberry pi and/or a tiny VPS.

**Reply by balzack** [reply]
@sreist let me know how it works when you have it up and running. I've tested a fair amount with a raspberry pi and a vps, but I primarily use a nuc with a bunch of other services.
